bindings/python/scanners.i

```diff
@@ -0,0 +1,33 @@
+%include "common.i"
+%include "fsm.i"
+
+%{
+#include <pire/scanners/multi.h>
+using Pire::Scanner;
+%}
+
+
+%nodefaultctor BaseScanner;
+%inline {
+    class BaseScanner {
+    };
+}
+
+
+class Scanner : public BaseScanner {
+public:
+
+    Scanner();
+
+    Scanner(Fsm&);
+
+    size_t Size() const;
+
+    bool Empty() const;
```

Comment:
Could you implement `Empty()` in `BaseScanner` only? The implementation would check `Size() == 0`.